doc.ddoc

```diff
@@ -67,6 +19,7 @@ $(SCRIPTLOAD js/cssmenu.js)
 </html>
 _=
 
+ROOT = .
```

Comment:
Maybe?

Anyway, this is just a nitpick. But a prefix with a trailing slash is a common pattern, e.g. the `--prefix` parameter to `./configure` scripts.